platform/platform-paxweb-jettyconfig/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -10,19 +10,58 @@
  * <http://www.gnu.org/licenses/lgpl.html>.
  *
  **/ -->
-<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+<blueprint xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+           xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
 
-
-    <bean id="securityJavaSubjectFilter"
-          class="org.codice.ddf.pax.web.jetty.SecurityJavaSubjectFilter"/>
+    <bean id="responseFilter" class="org.codice.ddf.pax.web.jetty.ResponseFilter">
+        <cm:managed-properties
+                persistent-id="org.codice.ddf.platform.response.filter.ResponseHeaderConfig"
+                update-strategy="container-managed"/>
+    </bean>
 
     <bean id="executor" class="java.util.concurrent.Executors" factory-method="newSingleThreadScheduledExecutor">
         <argument ref="securityJavaSubjectThreadFactory"/>
     </bean>
 
     <bean id="filterInjector" class="org.codice.ddf.pax.web.jetty.FilterInjector" init-method="init"
```

Comment:
❗️ Honestly if this possible to do this through pax web and just export Filter as a service we wouldn't even need to have a `FilterInjector` at all. If we can't I think that it might make sense to have this be plugable with other filters

Reply:
All of the filters that were moved into this bundle are security filters that would put the system at risk if they are turned off. I don't see any reason to leave them as optional. The pax web way of attaching other filters "should" work and we can test that out, but I don't think it is necessary for this change.

Reply:
I'm not sure why we would move these and not other security filters if we want them all in one place. `CsrfFilter` and `ResponseFilter` should be fine exporting `SecurityFilter` service because I don't think they care about ThreadContext.
Could be done in another ticket but just noting, FilterInjector is either not necessary or should be plugable for downstream projects (depending on pax web).

Reply:
There aren't any other security filters that aren't specifically part of the Authentication/Authorization path. All the ones that were real servlet filters were moved in this PR.